style.css

```diff
@@ -0,0 +1,111 @@
+*{
+	padding: 0;
+	margin: 0;
+	box-sizing: border-box;
+}
+
+.header {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	height: 65px;
+	padding: 14px;
+}
+
+.side-bar {
+	height: 100%;
+	width: 16%;
+	background-color: lightgrey;
+	overflow-y: hidden;
+}
+
+.main-content {
+	display: flex;
+	height: calc(100vh - 68px);
+	background-color: #e6f6c6;
+}
+
+@media (max-width: 765px) {
+	.side-bar {
+		display: none;
```

Comment:
Please! you may even try to fix your media query syntax for stylintrc check to pass